### DISCUSSION TITLE
Option to configure RELEASE_ALL=true without body/subject comment

### DISCUSSION THREAD

**Discussion by jprosevear** [TanStack/config#27] [General]
I prefer to keep library packages versioned in lockstep even if a specific package has no changes.  Is it reasonable to add a configuration option so that the already set RELEASE_ALL flag can be defaulted to true?

**Comment by jprosevear** [comment]
@lachlancollins I can write this if you are onboard with it

**Reply by lachlancollins** [reply]
Yeah that definitely seems reasonable, go for it!

BTW, are you using @tanstack/config in an open-source repo? I would love to have a look (since it's still kind of a "beta" internal tool).

**Reply by jprosevear** [reply]
#68 available for testing.

@lachlancollins we have been open sourcing a formerly internal framework we use to build MVPs; https://github.com/rhino-project/rhino-project and I needed

We've been using react-query and react-table since early 2020 and needed pnpm monorepo based build tooling (and our framework uses vite after converting from c-r-a) so @tanstack/config fit.  I prefer the vite lib setup to the initial try of tsup because we are still converting stuff to typescript.

Our backend is rails based and I hacked things to get it working the same way https://github.com/rhino-project/rhino-project/blob/beta/scripts/publish.js to publish to ruby gems.   With callbacks or export of some key functions after refactor, a lot of this hacky code could go away.
